### DISCUSSION TITLE
Query related to para

### DISCUSSION THREAD

**Discussion by HarishMurugancoolz** [Erudika/scoold#275] [Q&A]
unable to download the zip file from para using the export

**Comment by albogdano** [comment]
Do you host Para yourself? Is it behind a proxy, like nginx?

**Reply by HarishMurugancoolz** [reply]
yes , I am hosting the para as a docker container, I tried to use the ParaClient python library but not successful in getting the data. We are currently working on POC which involves Paraio and Scoold, hence wanted this information on how to import and export data from paraio. Would be great if you can provide some information on this. Thank you in advance

**Comment by albogdano** [comment]
What errors do you see in the Para logs? I can't say much about your case unless I see the logs.
